README.md

```diff
@@ -52,12 +57,15 @@ pod 'MiniApp'
 
 # Configuration
 
-In your project configuration .plist you should add 2 custom string target properties :
+In your project configuration .plist you should add 3 custom string target properties :
 
 - `RASApplicationIdentifier` - to set your MiniApp host application identifier
 - `RASProjectSubscriptionKey` - to set your MiniApp subscription key
 - `RMAAPIEndpoint` - to provide your own Base URL for API requests
 
+And a custom boolean target property:
```

Comment:
This boolean property is optional, right ? Not mandatory?

Reply:
yes, as it is only a boolean I thought that not providing it could be assimilated to false, I could mention it in documentation. By the way it's not an error to provide one to `false` if you mistakenly think it is mandatory :)